cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,15 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.3.2 <http://docs.cask.co/cdap/3.3.2/index.html>`__
+=============================================================
+
+Bug Fixes
+---------
+- `CDAP-5019 <https://issues.cask.co/browse/CDAP-5019>`__ - Fixed an issue with
+  launching Spark jobs on Cloudera Manager clusters when using Spark 1.4 or greater.
```

Comment:
The first question: Is this a CDH issue, or a CM issue? Is it any CDH cluster, both managed and not managed with CM, or is it restricted to CDH cluster managed with CM?

We can call it "CDH clusters managed with Cloudera Manager". See http://docs.cask.co/cdap/3.3.1/en/admin-manual/installation/packages.html.

But if it is really CDH clusters, then we should say:
"Cloudera Data Hub clusters when using Spark 1.4 or greater."